politeiawww/cmd/piwww/piwww.go

```diff
@@ -45,62 +45,67 @@ type piwww struct {
-	Votes           VotesCmd           `command:"votes" description:"(public) get the vote tally for a proposal"`
-	VoteResults     VoteResultsCmd     `command:"voteresults" description:"(public) get vote results for a proposal"`
-	VoteSummaries   VoteSummariesCmd   `command:"votesummaries" description:"(public) retrieve the vote summary for a set of proposals"`
-	VoteInventory   VoteInventoryCmd   `command:"voteinventory" description:"(public) retrieve the tokens of all public, non-abandoned proposal separated by vote status"`
-
-	// Commands
-	ChangePassword     shared.ChangePasswordCmd `command:"changepassword" description:"(user)   change the password for the logged in user"`
-	ChangeUsername     shared.ChangeUsernameCmd `command:"changeusername" description:"(user)   change the username for the logged in user"`
-	EditUser           EditUserCmd              `command:"edituser" description:"(user)   edit the  preferences of the logged in user"`
-	Help               HelpCmd                  `command:"help" description:"         print a detailed help message for a specific command"`
+	VoteAuthorize   voteAuthorizeCmd   `command:"voteauthorize" description:"(user) authorize a proposal vote (must be proposal author)"`
+	VoteStart       voteStartCmd       `command:"votestart" description:"(admin) start the voting period on a proposal"`
+	VoteStartRunoff voteStartRunoffCmd `command:"votestartrunoff" description:"(admin)  start a runoff using the submissions to an RFP"`
+	VoteBallot      voteBallotCmd      `command:"voteballot" description:"(public) cast ballot of votes for a proposal"`
+	Votes           votesCmd           `command:"votes" description:"(public) get the vote tally for a proposal"`
+	VoteResults     voteResultsCmd     `command:"voteresults" description:"(public) get vote results for a proposal"`
+	VoteSummaries   voteSummariesCmd   `command:"votesummaries" description:"(public) retrieve the vote summary for a set of proposals"`
+	VoteInventory   voteInventoryCmd   `command:"voteinventory" description:"(public) retrieve the tokens of all public, non-abandoned proposal separated by vote status"`
+	// XXX www vote routes
+	VoteDetails  voteDetailsCmd  `command:"votedetails" description:"(public) get the details for a proposal vote"`
+	VoteStatus   voteStatusCmd   `command:"votestatus" description:"(public) get the vote status of a proposal"`
+	VoteStatuses voteStatusesCmd `command:"votestatuses" description:"(public) get the vote status for all public proposals"`
+
+	// User commands
+	UserDetails        userDetailsCmd          `command:"userdetails" description:"(public) get the details of a user profile"`
+	UserEdit           userEditCmd             `command:"useredit" description:"(user) edit the preferences of the logged in user"`
+	UserNew            userNewCmd              `command:"usernew" description:"(public) create a new user"`
```

Comment:
Can we put the ordering as:
```
UserNew
UserEdit
UserDetails
```

This follows the same convention as the other routes.  Routes that write data are listed first.  Routes that read data are listed afterwards.